app/services/katello/pulp/rpm.rb

```diff
@@ -30,7 +30,7 @@ def provides
       end
 
       def files
-        if pulp_facts['files']
+        if pulp_facts['files'] && pulp_facts['files'].present?
```

Comment:
It was actually a case where 'files' exists; however, it is an empty/blank hash.